packages/opencensus-exporter-zpages/package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Did you miss an update to the files section of this file?

Reply:
Yes. Good catch